crates/iceberg-catalog/src/config.rs

```diff
@@ -59,7 +59,6 @@ fn get_config() -> DynAppConfig {
     // Fail early if the base_uri is not a valid URL
     config.s3_signer_uri_for_warehouse(WarehouseIdent::from(uuid::Uuid::new_v4()));
     config.base_uri_catalog();
-    config.base_uri_management();
```

Comment:
why remove this?

Reply:
We don't need it - I removed the whole method. I don't like that we need catalog either - its only required because of the design of the config endpoint

Reply:
I think ideally we would get rid of the base url altogether.
We might consider parsing it from the URL or the X-forwarded-for header.